index.html

```diff
@@ -0,0 +1,17 @@
+<html>
+  <header>
```

Comment:
```suggestion
  <head>
```
`stylesheet`の`link`,`meta`,`title`などのタグを囲むタグは`header`ではなく`head`ですね